src/Assembly.js

```diff
@@ -116,10 +115,26 @@ class Records {
 @observer
 class Assembly extends React.Component {
   @observable currentPage = Home
-  @observable coordinator = null
+  @observable coordinator = {
+    patients: [
+      {
+        med_report_status: 'notreported',
+        id: "pete",
+        took_medication: 'Yes',
+        firstname: "Peter",
+        lastname: "Campbell",
+        phone: 15304120086,
+        treatment_start_date: null,
+        last_repored_date: null,
+        side_effects: ["Nausea", "Redness"],
```

Comment:
Based on the schema above, these will be mapped via boolean values. Are we capturing that in the coordinator interface?